bin/gen-vcd

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+### Simple helper script to run a verilog file using verilator and generate
+### VCD dumps. Used by our testing framework. Should probably *not* be used
+### in production designs.
+
+set -eux
```

Comment:
What do these flags do?

Reply:
Some class options to build [robust shell scripts](https://sipb.mit.edu/doc/safe-shell/).